components/org.wso2.carbon.identity.organization.management.service/src/main/java/org/wso2/carbon/identity/organization/management/service/OrganizationManagerImpl.java

```diff
@@ -717,6 +733,7 @@ private void validateOrganizationPatchOperations(List<PatchOperation> patchOpera
                     throw handleClientException(ERROR_CODE_SUPER_ORG_RENAME, organizationId);
                 }
                 validateOrganizationNameField(value);
+                newOrgName = value;
```

Comment:
No need to maintain another variable right ?
```suggestion
            Organization organization = organizationManagementDAO.getOrganization(organizationId);
            validateOrgNameUniquenessAmongSiblings(organization.getParent().getId(), newOrgName);
```

Reply:
Fixed by - https://github.com/wso2-extensions/identity-organization-management-core/pull/29/commits/9153863550364ed9286154d0fda7bf0d75ac820b